lint/tests/test_lint.py

```diff
@@ -153,6 +153,58 @@ def test_lint_failing(capsys):
     assert err == ""
 
 
+def test_ref_existent_relative(capsys):
+    with _mock_lint("check_path") as mocked_check_path:
```

Comment:
I chose to add the tests to this file because the behavior is dependent on the
state of the file system, and these tests are already structured to provide the
necessary "dummy" repo.